src/main/java/org/opendatakit/aggregate/client/permissions/AccessConfigurationSheet.java

```diff
@@ -82,7 +80,9 @@
     userType = new ArrayList<String>();
     userType.add(ACCOUNT_TYPE_ODK);
     userType.add(ACCOUNT_TYPE_GOOGLE);
-  };
+  }
+
+  ;

```

Comment:
By reformatting and exposing these unnecessary semicolons—I count ten—you take on the responsibility of deleting them. 😀

Reply:
I wonder how on earth did I miss those! Removing them :)